backend/pokemon/models.py

```diff
@@ -1,12 +1,10 @@
-from django.conf import settings
+# from django.conf import settings
 from django.db import models
 
+
 class Pokemon(models.Model):
-    id = models.AutoField(primary_key=True)
+    pokemon_id = models.CharField(max_length=200, null=True)
```

Comment:
Why do you need an id saved as char?

Reply:
Actually, I don't need it, I will resolve this saving as Integer.